static/scripts/newblocks.js

```diff
@@ -739,6 +739,28 @@ Blockly.JavaScript["check_fitness"] = function (block) {
   return [code, Blockly.JavaScript.ORDER_FUNCTION_CALL];
 };
 
+
+Blockly.JavaScript["plotting_one_value"] = function (block) {
+  var variable_ = Blockly.JavaScript.valueToCode(
+    block,
+    "yValue",
+    Blockly.JavaScript.ORDER_ATOMIC
+  );
+  var code = "plot(";
+  code += variable_ + ",";
+  variable_ = Blockly.JavaScript.valueToCode(
```

Comment:
I think it'd look better if you fetch the field variables first and generate the code as a second step :)